src/game/scenes/Game.js

```diff
@@ -16,8 +18,14 @@ export default class Game extends Phaser.Scene {
     const centerY = this.scale.height / 2;
 
     const customObject = new Animatronic(this, AnimatronicsNames.BB, centerX, centerY);
+    const cupPart1 = new Cup(this, this.scale.width, this.scale.height, 3, 2, Math.PI/2);
+    const cupPart2 = new Cup(this, this.scale.width, this.scale.height, 2, 1.365, 0);
+    const cupPart3 = new Cup(this, this.scale.width, this.scale.height, 1.5, 2, Math.PI/2*3);
```

Comment:
made it as 3 lines of code, feel free to change it if you want to.

Reply:
Should be a single object as I said in my review